contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -318,6 +356,10 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
         // We've added VAIController as a borrowed token list check for seize
         ensureListed(market);
 
+        if (liquidator != address(0) && whitelistedExecutors[liquidator]) {
+            return uint256(Error.NO_ERROR);
```

Comment:
Should we still call `updateVenusSupplyIndex`, `distributeSupplierVenus` to stop accruing XVS for the supply that no longer exists?